prompting/tasks/task_registry.py

```diff
@@ -67,8 +69,7 @@ class TaskRegistry(BaseModel):
         # ),
         TaskConfig(
             task=WebRetrievalTask,
-            # TODO: Increase probability after v2.9.0.
-            probability=0.03,
+            probability=0.1,
```

Comment:
Please check the comments to the previous PR to pre-staging. Increasing prob will lead to rate limit